### ISSUE TITLE
Consider adding .py to usage in command line documentation

### ISSUE THREAD

**Issue by ssnn-airr** [immcantation/changeo#176]
**[Original report](https://bitbucket.org/kleinstein/changeo/issues/176) by William Lees (Bitbucket: [williamlees](https://bitbucket.org/williamlees), GitHub: [williamlees](https://github.com/williamlees)).**

----------------------------------------

Although it is clear in the examples, the command line documentation does not state that .py needs to be added to the end of each of the Changeo commands - example below. I’ve just been approached by someone in my department who spent quite a long time, unsuccessfully, trying to run the commands in the Docker container and it’s possible that others might have hit the same brick wall.

Best wishes

William

```
usage: ConvertDb [--version] [-h]  ...
```

‌

**Comment by ssnn-airr** [comment]
**Original comment by Jason Vander Heiden (Bitbucket: [javh](https://bitbucket.org/javh), GitHub: [javh](https://github.com/javh)).**

----------------------------------------

Interesting… That’s from the online docs, yes? Looks like there’s some quirk to how the Sphinx docs are generating the command line help. It looks correct from the commandline.

I’ll see if I can figure it out. Thanks, @{557058:cdb12031-cfb0-443b-b6c5-95ba758d58e9} .

```
$ ConvertDb.py -h

usage: ConvertDb.py [--version] [-h]  ...

Parses tab delimited database files

help:
  --version   show program's version number and exit
  -h, --help  show this help message and exit

subcommands:
              Database operation
    airr      Converts input to an AIRR TSV file.
    changeo   Converts input into a Change-O TSV file.
    fasta     Creates a fasta file from database records.
    baseline  Creates a specially formatted fasta file from database records
              for input into the BASELINe website. The format groups clonally
              related sequences sequentially, with the germline sequence
              preceding each clone and denoted by headers starting with ">>".
    genbank   Creates files for GenBank/TLS submissions.

output files:
    airr
        AIRR formatted database files.
    changeo
        Change-O formatted database files.
    sequences
        FASTA formatted sequences output from the subcommands fasta and clip.
    genbank
        feature tables and fasta files containing MiAIRR compliant input for tbl2asn.

required fields:
    sequence_id, sequence, sequence_alignment, junction, v_call, d_call, j_call, 
    v_germline_start, v_germline_end, v_sequence_start, v_sequence_end, 
    d_sequence_start, d_sequence_end, j_sequence_start, j_sequence_end 

optional fields:
    germline_alignment, c_call, clone_id 
```

‌

**Comment by ssnn-airr** [comment]
**Original comment by William Lees (Bitbucket: [williamlees](https://bitbucket.org/williamlees), GitHub: [williamlees](https://github.com/williamlees)).**

----------------------------------------

Yes. From the online docs. Thanks!

**Comment by ssnn-airr** [comment]
**Original comment by Jason Vander Heiden (Bitbucket: [javh](https://bitbucket.org/javh), GitHub: [javh](https://github.com/javh)).**

----------------------------------------

Fixed in 81a8f91.  Just now getting around to a new release - change should be on the landing page in a day or two.
